docs/website/docs/tutorial/filesystem.md

```diff
@@ -0,0 +1,276 @@
+```
+
+## 5. Exploring the data
+
+Now that the pipeline has run successfully, let's explore the data loaded into DuckDB. `dlt` comes with a built-in browser application that allows you to interact with the data. To enable it, run the following command:
+
+```sh
+pip install streamlit
+```
+
+Next, run the following command to start the data browser:
+
+```sh
+dlt pipeline s3_to_duckdb show
+```
+
+The command opens a new browser window with the data browser application. `s3_to_duckdb` is the name of the pipeline defined in the `filesystem_pipeline.py` file.
+You can explore the loaded data, run queries and see some pipeline execution details.
+
+## 6. Appending, replacing, and merging loaded data
+
+Try running the pipeline again with `python filesystem_pipeline.py`. You will notice that
+all the tables have data duplicated. This happens because by default, `dlt` appends the data to the destination table. It is very useful, for example, when you have daily data updates and you want to ingest them. In `dlt` you can control how the data is loaded into the destination table by setting the `write_disposition` parameter in the resource configuration. The possible values are:
+- `append`: Appends the data to the destination table. This is the default.
+- `replace`: Replaces the data in the destination table with the new data.
+- `merge`: Merges the new data with the existing data in the destination table based on the primary key.
+
+To specify the `write_disposition` you can set it in `pipeline.run` command:
+
+```py
+import dlt
+from dlt.sources.filesystem import filesystem, read_parquet
+
+parquet_resource = filesystem(file_glob="**/*.parquet") | read_parquet()
+pipeline = dlt.pipeline(pipeline_name="s3_to_duckdb", dataset_name="parquet_data", destination="duckdb")
+info = pipeline.run(parquet_resource, write_disposition="replace")
+print(info)
+```
+
+You can learn more about write_disposition in the [Write dispositions section](../general-usage/incremental-loading#the-3-write-dispositions).
+
+## 7. Loading data incrementally
+
+When loading data from files, you often only want to load files that have been modified. `dlt` makes this easy with [incremental loading](../general-usage/incremental-loading). To load only modified files you can use `apply_hint`
+function:
+
+```py
+import dlt
+from dlt.sources.filesystem import filesystem, read_parquet
+
+files_resource = filesystem(file_glob="**/*.parquet")
+files_resource.apply_hints(incremental=dlt.sources.incremental("modification_date"))
+parquet_resource = files_resource | read_parquet()
+pipeline = dlt.pipeline(pipeline_name="s3_to_duckdb", dataset_name="parquet_data", destination="duckdb")
+info = pipeline.run(parquet_resource, write_disposition="replace")
+print(info)
+```
+
+As you can see the incremental hint is applied to the `filesystem` resource before `read_parquet` transformer, because
+we want to apply the incremental loading to filter the loaded files.
```

Comment:
this is also less verbose than Anton's tutorial - should we include the full instructions?